components/brave_wallet_ui/components/extension/confirm-transaction-panel/confirm-zcash-transaction-panel.tsx

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
Lets rename this file to use `snake_case` convention.

`confirm_zcash_transaction_panel.tsx`